frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -53,6 +58,15 @@ function getValidationError(operator: PropertyOperator, value: any, property?: s
         }
         return message
     }
+
+    // URL properties shouldn't allow for spaces in value
+    if (
+        property &&
+        property.toLowerCase().includes('url') &&
```

Comment:
Does it work for arbitrary string properties? For example, if I use a custom property `url_type` with a value `freshly created`, won't it also return a validation error? A better way could be testing only Posthog's reserved properties (e.g., starting from $) or using property definitions if possible.

Reply:
That's a great idea. I could check it's a reserved property so that it starts with $ + also has url